### PR TITLE
Adding my lesson

### PR DESCRIPTION
**Pull request by KevRozmiarek**
This is for peer feedback

### CODE REVIEW

**Comment by sebkopf** [comment]
please use link: <a href='https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-KevRozmiarek/binder-R?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-KevRozmiarek%26branch%3Dweek1%26urlpath%3Drstudio'></a>

File: Classic Momentum.Rmd [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Classic Momentum"
+subtitle: "How to smack things together"
```
Comment (sebkopf) [inline Classic Momentum.Rmd:3]: great title!

File: Classic Momentum.Rmd [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Because momentum is **conserved** in elastic collisions, momentum of a system is the summation of is parts. Take for example two cars moving in the same direction with masses $m_{1}$ and $m_{2}$ and velocities $v_{1}$ and $v_{2}$ respectively. The total mass of the system is then:
+
+$$
+\rho_{total}=m_{1}v_{1}+m_{2}v_{2}=\rho_{1}+\rho_{2}
+$$
+Because of this conservation, we can explore the concept of many differing colliding objects seen below in our example.
+
+# Example
+
+Two cars are barreling towards another car in a future ellastic collision. There masses and velocities are listed below. Lets use R to calculate the total mass of the system!
```
Comment (sebkopf) [inline Classic Momentum.Rmd]: Their?

File: Classic Momentum.Rmd [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+```{r}
```
Comment (sebkopf) [inline Classic Momentum.Rmd]: add `, message=FALSE` to suppress the info messages

File: Classic Momentum.Rmd [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+library(tidyverse)
+
+# session info
+masses <- c(300, 400, 600)
+velocties <- c(20, 10, -15)
```
Comment (sebkopf) [inline Classic Momentum.Rmd:54]: are you sure the students are going to understand that velocities are vectors and therefore have direction? and that this is movement in one dimension

File: Classic Momentum.Rmd [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+total_momentum <- sum(masses*velocties)
+print(total_momentum)
+
+```
+
+And there you go! The total momuntum of the system is 1000! More specifically it is $kg⋅m/s$. This implies that though to one car is hitting the other two. The three will still continue onwards in the direction of the original two.... though perhaps slower than before.
```
Comment (sebkopf) [inline Classic Momentum.Rmd:61]: the direction aspect might make more sense if you discuss directionality in a little more detail

File: Classic Momentum.Rmd [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+# Questions
+
+1. You just got your BB gun for Christmas but all is for naught because an astroid is bearing down on Earth! The astroid is $1  \times 10^{15} kg$ and moving at $2 \times 10^{4} m/s$. Fortunatly you have quite a few BBs which weigh $0.0002 kg$ and you can fire them at $60 m/s$. How many BBs will it take to stop the astroid and save Earth?
```
Comment (sebkopf) [inline Classic Momentum.Rmd]: give instructions to the students that help them understand how they are supposed to solve this: 
 - please show all steps
- use R to do this calculation

File: Classic Momentum.Rmd [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand that momentum is conserved in collisions.
+
+2. Understand the difference between **elastic** and **inelastic** collisions
```
Comment (sebkopf) [inline Classic Momentum.Rmd:22]: maybe include a question at the end that tests this understanding

**Review by sebkopf — CHANGES_REQUESTED** [review]

File: Classic Momentum.Rmd (hunk 1 above)
Comment (KevRozmiarek) [reply Classic Momentum.Rmd:3]: Thanks :)

File: Classic Momentum.Rmd [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+3. Be able to balence and compute the momentum on objects.
```
Comment (vaga3461) [inline Classic Momentum.Rmd]: very nit-picky, but should it be momentum *of* objects?

File: Classic Momentum.Rmd [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+# Background
+
+Momentum is a quality of everything! It is the quantity of motion of a moving body, measured as a product of its mass and velocity. Momentum by itself isn't all that interesting, only when it comes to collisions! Let's talk about two different kinds of collisions first.
+
+**Elastic collision:** An elastic collision is an encounter between two bodies in which the total momentum and kinetic energy of the two bodies remains the same. In an ideal, perfectly elastic collision, there is no net conversion of kinetic energy into other forms such as heat, noise, or potential energy.
```
Comment (vaga3461) [inline Classic Momentum.Rmd:30]: could add a quick review of what kinetic energy is here (unless assuming students have already covered that topic, which would probably be the case)

File: Classic Momentum.Rmd [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+**Inelastic Collision:** An inelastic collision, in contrast to an elastic collision, is a collision in which momentum and kinetic energy is not conserved due to the action of internal friction.
```
Comment (vaga3461) [inline Classic Momentum.Rmd]: same with internal friction - could say just a couple words defining what that is

File: Classic Momentum.Rmd (hunk 5 above)
Comment (vaga3461) [reply Classic Momentum.Rmd:61]: agreed, or just a quick note that a result with the opposite sign would indicate a change in direction

**Review by vaga3461 — CHANGES_REQUESTED** [review]
Question 1 is brilliant

File: Classic Momentum.Rmd (hunk 8 above)
Comment (splazatorres) [reply Classic Momentum.Rmd]: also nit picky but it says balence istead of balance

File: Answer Key for Examples.Rmd [hunk 11]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Answer Key for Examples"
+author: "Kevin Rozmiarek"
+date: "9/1/2020"
+output: html_document
+---
+
+# Questions
+
+1. You just got your BB gun for Christmas but all is for naught because an astroid is bearing down on Earth! The astroid is $1  \times 10^{15} kg$ and moving at $2 \times 10^{4} m/s$. Fortunatly you have quite a few BBs which weigh $0.0002 kg$ and you can fire them at $60 m/s$. How many BBs will it take to stop the astroid and save Earth?
```
Comment (splazatorres) [inline Answer Key for Examples.Rmd:10]: Love this!

File: Classic Momentum.Rmd (hunk 10 above)
Comment (splazatorres) [reply Classic Momentum.Rmd]: yup

File: Classic Momentum.Rmd (hunk 8 above)
Comment (splazatorres) [reply Classic Momentum.Rmd]: also maybe specify which kind of collisions (elastic or inelastic? / if you do both then cool, just ignore this) you want to calculate the momentum for.

File: Classic Momentum.Rmd (hunk 5 above)
Comment (splazatorres) [reply Classic Momentum.Rmd:61]: ^^^^

File: Classic Momentum.Rmd (hunk 6 above)
Comment (splazatorres) [reply Classic Momentum.Rmd]: Also maybe let them know as a reminder that this is based on an elastic collision

**Review by splazatorres — COMMENTED** [review]
Awesome notebook. Really liked it! It's very creative and fun. The only thing I'd comment is to maybe help the (hypothetical) students a little bit more with question 1 so they don't get lost.

**Pull request merged**